recipes/hyperscan/all/patches/fix-cmake.patch

```diff
@@ -220,35 +220,216 @@ index 8bc6077..579a7b4 100644
                  )
          endif (BUILD_AVX512VBMI)
          add_library(hs_exec_common_shared OBJECT
-diff --git chimera/CMakeLists.txt chimera/CMakeLists.txt
-index 1cd66a3..ebb3b49 100644
---- chimera/CMakeLists.txt
-+++ chimera/CMakeLists.txt
-@@ -44,6 +44,6 @@ if (NOT WIN32)
+diff --git a/chimera/CMakeLists.txt b/chimera/CMakeLists.txt
+index 1cd66a3..795f5f2 100644
+--- a/chimera/CMakeLists.txt
++++ b/chimera/CMakeLists.txt
+@@ -18,6 +18,7 @@ SET(chimera_SRCS
+     ${chimera_HEADERS}
+     ch_alloc.c
+     ch_alloc.h
++    ch_pcre_bridge.c
+     ch_compile.cpp
+     ch_database.c
+     ch_database.h
+@@ -27,9 +28,13 @@ SET(chimera_SRCS
+     ch_scratch.c
+ )
+ 
++if(CMAKE_CXX_COMPILER_ID MATCHES "MSVC")
++   set_source_files_properties(ch_pcre_bridge.c PROPERTIES COMPILE_FLAGS "/Gd")
++endif()
++
+ add_library(chimera STATIC ${chimera_SRCS})
+-add_dependencies(chimera hs pcre)
+-target_link_libraries(chimera hs pcre)
++add_dependencies(chimera hs PCRE::libpcre)
++target_link_libraries(chimera hs PCRE::libpcre)
+ 
+ install(TARGETS chimera DESTINATION ${CMAKE_INSTALL_LIBDIR})
+ 
+@@ -44,6 +47,6 @@ if (NOT WIN32)
      set(PRIVATE_LIBS "${PRIVATE_LIBS} -L${LIBDIR} -lpcre")
  
      configure_file(libch.pc.in libch.pc @ONLY) # only replace @ quoted vars
 -    install(FILES ${CMAKE_BINARY_DIR}/chimera/libch.pc
 +    install(FILES ${CMAKE_CURRENT_BINARY_DIR}/libch.pc
```

Comment:
There's no reason to fix this install command since the file is deleted in the `conanfile.py` package step anyways. Can you just comment out the `install` command here?

Reply:
True, I will fix it as soon as possible. Thanks !

Reply:
Fixed